tests/missing/systemd/journal/__init__.py

```diff
@@ -46,6 +49,9 @@ def add_match(self, *args, **kwargs):
     def seek_cursor(self, cursor):
         raise RuntimeError
 
+    def seek_tail(self):
+        raise RuntimeError
+
```

Comment:
Probably want to mock `close` as well, with `pass`.

Reply:
Will do.